README.md

```diff
@@ -1,5 +1,4 @@
-[![PyPI Latest Release](https://img.shields.io/pypi/v/fawltydeps.svg)](https://pypi.org/project/fawltydeps/) [![Supported Python versions](https://img.shields.io/pypi/pyversions/fawltydeps.svg)](https://pypi.org/project/fawltydeps/) [![Licence](https://img.shields.io/pypi/l/fawltydeps.svg)](https://pypi.org/project/fawltydeps/)
-
+[![PyPI Latest Release](https://img.shields.io/pypi/v/fawltydeps.svg)](https://pypi.org/project/fawltydeps/) [![Supported Python versions](https://img.shields.io/pypi/pyversions/fawltydeps.svg)](https://pypi.org/project/fawltydeps/) ![Build](https://img.shields.io/github/actions/workflow/status/tweag/fawltydeps/ci.yaml) [![Licence](https://img.shields.io/pypi/l/fawltydeps.svg)](https://pypi.org/project/fawltydeps/)
```

Comment:
Nit: Maybe one shield per line (unless it screws up the formatting) so that these diffs become more readable?

Reply:
It screws up formation :( 
<img width="308" alt="Screenshot 2023-03-15 at 10 11 42" src="https://user-images.githubusercontent.com/27200848/225261694-70ce8c1d-6fbf-4a1d-9338-64a1b6eca8ec.png">